srcpkgs/rawtherapee/patches/glibc-2.36.patch

```diff
@@ -0,0 +1,379 @@
+Thumbnail::readAEHistogram and Thumbnail::writeAEHistogram have been
```

Comment:
could you put the source of the patch before this line?

Reply:
The patch for `Thumbnail::readAEHistogram` and `Thumbnail::writeAEHistogram` does not have the external source (if you mean url or something like that).

By the time they fixed build with glibc >=2.34, these functions were already gone. Since upstream did not backport this fix for 5.8, I have adapted it myself.

I take it my intent behind the message in the first patch is not understood, so I will try to correct it.